docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,185 @@
+    - *Tip*: Flood the geometry with a very low value 0.1 - 0.2.
 
-<figure>
-  <img src="images/paint_tool.png" caption="AdonisFX Paint Tool"> 
-  <figcaption>Figure 2: AdonisFX Paint Tool</figcaption>
-</figure>
+    - *Tip*: It may be desirable to leave this map weight to 1 as a starting point and tweak some areas later on as the results of the skin simulation are seen.
+
+- **Slide Constraints**: Weight to modulate the correction applied to the vertices to keep them at a constant distance to the reference mesh sliding along the reference surface. In the example of a biped or quadruped, it is recommended to set a value of 1.0 on the scapulas, shoulders, elbows and knees and an overall value of 0 on the rest of the body.
+
+    - *Tip*: Smooth the borders by using the Smooth-Flood combination to make sure the simulation doesn’t find hard edges as this could create odd looking wrinkles (unless that’s something you are looking for!).
+
+- **Compression Resistance**: Force to correct the edge lengths if the current length is smaller than the rest length. A higher value represents higher correction. At value 1 the points in the geometry will try to stay as close as possible to their original position.
+
+    - *Tip*: It may be desirable to leave this map weight to 1 as a starting point and tweak some areas later on as the results of the skin simulation are seen.
+    
+    - *Tip*: Reducing the value of the weight in some areas will contribute to getting rid of unwanted wrinkles or possible artifacts in the skin.
 
-Do the following to open the tool:
+- **Stretching Resistance**: Force to correct the edge lengths if the current length is greater than the rest length. A higher value represents higher correction.
 
-  1. Select the mesh with the Skin deformer applied to.
-  2. Press the paint tool ![paint tool](images/adn_paint_tool.png) shelf button or go to AdonisFX menu > Paint Tool.
+    - *Tip*: It may be desirable to leave this map weight to 1 as a starting point and tweak some areas later on as the results of the skin simulation are seen.
 
-You can select the attribute to paint from the combo box exposed in the UI:
+    - *Tip*: Smooth the borders by using the smooth-Flood combination to make sure the simulation doesn’t find hard edges as this could create odd wrinkles (unless that’s something you are looking for!).
 
-<figure style="margin-left:30%;" markdown> 
-  ![Pain Tool Skin Attributes example](images/paint_tool_skin_attributes.png) 
-  <figcaption> Figure 3: Paintable attributes listed in the UI. </figcaption>
+- **Global Damping**: Set global damping per vertex in the simulated mesh. The greater the value per vertex the more it will attempt to retain its previous position.
```

Comment:
```suggestion
- **Global Damping**: Set global damping per vertex in the simulated mesh. The greater the value per vertex is the more it will attempt to retain its previous position.
```

Reply:
Done!